.github/workflows/sandbox-deletion.yml

```
@@ -0,0 +1,41 @@
+          access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          region: eu-central-1
+
+      - name: Trigger AWS CodePipeline for PR Closure (non-merge)
+        if: env.merged == 'false'
+        uses: aws-actions/aws-codepipeline-action@v1
+        with:
+          pipeline-name: sandbox-deletion
+          access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          region: eu-central-1
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)